src/main/app/package-lock.json

```diff
@@ -1887,7 +1887,7 @@
     "@mapbox/jsonlint-lines-primitives": {
       "version": "2.0.2",
       "resolved": "https://registry.npmjs.org/@mapbox/jsonlint-lines-primitives/-/jsonlint-lines-primitives-2.0.2.tgz",
-      "integrity": "sha512-rY0o9A5ECsTQRVhv7tL/OyDpGAoUB4tTvLiW1DSzQGq4bvTPhNw1VpSNjDJc5GFZ2XuyOtSWSVN05qOtcD71qQ=="
+      "integrity": "sha1-zlblOfg1UrWNENZy6k1vya3HsjQ="
```

Comment:
Ik heb het idee dat deze niet gecommit moet worden. Dit moet toch sha512 zijn? graag even verifieren bij Bart of Ronnie

Reply:
Dat heb ik gedaan op uitdrukkelijk verzoek van Ronnie EN in overleg met Bart. ;-)